core/templates/domain/collection/collection-node.model.ts

```diff
@@ -39,14 +39,14 @@ export class CollectionNode {
 
   static create(
       collectionNodeBackendObject: CollectionNodeBackendDict):
-      CollectionNode {
+    CollectionNode {
```

Comment:
```suggestion
      CollectionNode {
```

Reply:
Doing this change gives linting issue-indentation error.